docs/guides/data/creating-a-cloud-app.md

```diff
@@ -0,0 +1,329 @@
+
+!!! tip
+    It is not necessary to use `dx-app-wizard`. All the necessary files and project directory structure can be created manually. However, `dx-app-wizard` provides a quick and easy way to get started. For more information, refer to the [Advanced App Tutorial](https://wiki.dnanexus.com/Developer-Tutorials/Advanced-App-Tutorial).
+
+All DNAnexus project applications will have the following structure:
+
+```
+dx-fastqc-example-app/
+├── dxapp.json
+├── resources/
+│   └── usr/
+│       └── bin/
+└── src/
+    └── dx-fastqc-example-app.sh
+```
+
+The `dxapp.json` file is a JSON file that contains metadata about the application we are writing that are needed to build and run the app on the DNAnexus Platform. Most notably, you will need to specify all of the inputs your app requires (both input files or any settings you can tune), output files, and other options such as the number of cores and memory required to run the tool. To see the full list of fields, refer to the [DNAnexus wiki](https://wiki.dnanexus.com/dxapp.json) guide on the application metadata.
+
+The `dx_fastqc_example_app.sh` file is a bash script is what will be executed when the application is run. Any executable binaries that accompany the application, such as other tools or scripts, are placed in the `resources` folder. From there, we can call the executable from within the app when it is run.
+
+## Creating the Project
+
+Start by running the `dx-app-wizard` command from your terminal.
+
+```bash
+$ dx-app-wizard
+```
+
+For our inputs, we will enter the following:
+
+```bash
+$ App Name: dx-fastqc-example-app
+...
+$ Title []: FastQC Example Application
+...
+$ Summary []: Uses FastQC to generate quality control reports on raw sequence data.
+...
+$ Version [0.0.1]: 0.0.1
+...
+$ 1st input name (<ENTER> to finish): bam_file
+$ Label (optional human-readable name) []: BAM File
+...
+$ Choose a class (<TAB> twice for choices): file
+$ This is an optional parameter [y/n]: n
+...
+$ 1st output name (<ENTER> to finish): fastqc_html
+$ Label (optional human-readable name) []: FastQC HTML Report
+$ Choose a class (<TAB> twice for choices): file
+
+$ 2nd output name (<ENTER> to finish): fastqc_zip
+$ Label (optional human-readable name) []: FastQC Zip File
+$ Choose a class (<TAB> twice for choices): file
+...
+$ Timeout policy [48h]: 48h
+...
+$ Programming language: bash
+...
+$ Will this app need access to the Internet? [y/N]: y
```

Comment:
Does this app really need access to the internet? I'm thinking we should change this to "N".

Reply:
I'll change it to "N", but I'll address it and include a paragraph/tip about it below.